compiler/z/codegen/OMRInstOpCodeProperties.hpp

```diff
@@ -5670,7 +5673,7 @@
    /* .minimumALS  = */ OMR_PROCESSOR_S390_Z900,
    /* .properties  = */ S390OpProp_IsStore |
                         S390OpProp_LongDispSupported |
-                        S390OpProp_SetsOperand3
```

Comment:
I am bit curious how this flag is used in the binary encoding.  As bunch of places, you replaced `setsOperand3` does not even use that when instruction was constructed. 
Looking at one of the constructor for RSInstruction [1], previously it was setting `lreg` as source register, but with this change, it will set it as target Register. 

[1]. https://github.com/eclipse/omr/blob/4553376e32c8fcdac64f2a2bfa3ae0ae02f7b467/compiler/z/codegen/S390Instruction.hpp#L2591-L2594